apps/frontend/src/libs/components/menu-item/styles.module.css

```
@@ -0,0 +1,31 @@
+.project-menu-item {
+	display: flex;
+	gap: 8px;
+	align-items: center;
+	padding: 11px 15px;
+	text-align: left;
+	cursor: pointer;
+	background-color: transparent;
+	border: none;
+	border-radius: 4px;
+}
+
+.project-menu-item:hover {
+	background-color: var(--color-background-hover);
+}
+
+.project-menu-item-text {
+	margin: 0;
+	font-family: Inter, sans-serif;
+	font-size: 16px;
+	font-weight: 500;
+	line-height: 1.2;
+}
+
+.project-menu-item--primary {
```

We don't use BEM in our project, so one dash will be enough
```suggestion
.project-menu-item-primary {
```

---

And it should go without project prefix